### ISSUE TITLE
HTTP Request DNS Problem

### ISSUE THREAD

**Issue by HakanDmr** [a85/POSTMan-Chrome-Extension#3759]
## App Details:
```
Postman for Windows
Version 5.3.2
win32 10.0.15063 / x64
```

## Issue Report:
Hi,
 When I try to send a request after the few other request Postman not sending my request. 
 Subsequently, I'm flush my dns cache with the below command and Postman started to working.

`ipconfig /flushdns`

I'm not sure about this is a bug or something about my computer.

Thanks.

**Comment by prashantagarwal** [comment]
@HakanDmr Thanks for reporting we will investigate this.

**Comment by ghost** [comment]
+1

I have added a subdomain. On Chrome everything works well but Postmen can't see new subdomain until I flush my DNS.

**Comment by BerkanYildiz** [comment]
Still an issue, had to restart postman and flush the dns cache.

**Comment by totoleo** [comment]
+10086

**Comment by codenirvana** [comment]
@totoleo We are having trouble replicating this at our end. Could you provide the steps to reproduce this behavior at our end?

**Comment by shamasis** [comment]
@HakanDmr @BerkanYildiz @totoleo - can you still reproduce this issue?

1. We could not reproduce this. DNS related network aspects have too many intermediate moving parts and hard to get original env reproduced

2. Postman app, by itself, has no DNS cache-ing whatsoever. It relies on whatever the OS/Network DNS decides.

Will close this issue after some inactivity time.

**Comment by BerkanYildiz** [comment]
Have not touched to Postman for so long, sorry. (I don’t normally do web dev)

But the bug is definitely not an issue of Postman and therefore should be closed, I suppose.

As stated, cleaning Windows’s DNS cache fixes it.

The problem is the library (I guess ?) Postman uses to handle all the web requests, etc..

Need to handle the DNS system yourself by doing the hostname resolution by itself, pass the IP instead of the actual hostname, and set the required hostname header to that hostname. 

To reproduce the issue, the subdomain must not be created first.
Then you have to try sending a get/post request to it so windows caches it.
Then the subdomain is added, but the Windows’s cache will make it fail/not found.

TL;DR: not a Postman issue.

**Comment by tsal** [comment]
> 2. Postman app, by itself, has no DNS cache-ing whatsoever. It relies on whatever the OS/Network DNS decides.

I'm not entirely sure this is (still) true.  I've flushed dscache on my mac, and net-internals in chrome, and resolution works fine for the OS and chrome - but not postman.

**Comment by enkelmedia** [comment]
I have the same issue =D

**Comment by enkelmedia** [comment]
Hmm...

Its totally related to Postman, i had the same issue, flushing dns, restarting the machine and checked that the domain resolved to the right server using cmd-ping.

Still did not work in Postman, but... I tried to fiddle with the settings and found something called "Use next generation URL processing" - turning this off solved the issue. When turning it back on the issue comes back. So it's for sure something caches in the background when this feature is activated.

![image](https://user-images.githubusercontent.com/1782524/96705006-07644a00-1395-11eb-86bf-403f9ee6951f.png)

**Comment by jakebanks** [comment]
> Hmm...
> 
> Its totally related to Postman, i had the same issue, flushing dns, restarting the machine and checked that the domain resolved to the right server using cmd-ping.
> 
> Still did not work in Postman, but... I tried to fiddle with the settings and found something called "Use next generation URL processing" - turning this off solved the issue. When turning it back on the issue comes back. So it's for sure something caches in the background when this feature is activated.
> 
> ![image](https://user-images.githubusercontent.com/1782524/96705006-07644a00-1395-11eb-86bf-403f9ee6951f.png)

It looks like this option has been removed :(

I do find that Chrome and Postman seem to have different DNS caching - last night Chrome was using its cache but Postman wasn't, now this morning I have the opposite problem.

Updating my Postman version worked, but I feel that it was just the application restart that did it (could have sworn I tried that already though  `¯\_(ツ)_/¯`)
